modules/core/src/main/java/org/apache/synapse/api/Resource.java

```diff
@@ -213,6 +217,18 @@ public void setProtocol(int protocol) {
         this.protocol = protocol;
     }
 
+    public Set<String> getInboundEndpointBindings() {
+        return inboundEndpointBindings;
+    }
+
+    public void addInboundEndpointBinding(String inboundEndpointName) {
```

Comment:
this method is not used anywhere, shall we remove

Reply:
Addressed in 89315e2 and cc02c4c